go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/anchore/grype
 
-go 1.18
+go 1.19
```

Comment:
Do the `1.19` updates need to happen as part of this PR?

Reply:
I think in this case it did, but it might had to do with how my local setup had to be updated midway through this PR. I was getting a lot of module related error messages about 1.18. Upgrading everything to 1.19 brought my module checks out of the red and allowed the status to pass 🟢 

Also it looks like we're getting close to EOLfor 1.18 with 1.20 around the corner so the fix seems pretty clear
```
Each major Go release is supported until there are two newer major releases. 
For example, Go 1.5 was supported until the Go 1.7 release, and Go 1.6 was supported until the Go 1.8 release. 
We fix critical problems, including [critical security problems](https://go.dev/security), in supported releases as needed by issuing minor revisions (for example, Go 1.6.1, Go 1.6.2, and so on).
```

Reply:
regardless of the change being needed, I'm ok with this being mixed in this PR given that the (unlikely) fallback PR is easy to craft.